src/openrct2/ride/ride.h

```diff
@@ -41,6 +41,7 @@
 #define MAX_RIDES                       255
 #define RIDE_ID_NULL                    255
 
+static const uint8      ride_adjacent_station_max_distance  = 5;
```

Comment:
This is effectively creating a local variable in each translation unit including this header.

Reply:
What would be the appropriate way of doing it?

Reply:
What is the intended use of it? This is `static const`, so I imagine it's not meant to be modified in runtime? If that's the case, perhaps you can go with a define. If you expect this will be changed in runtime, use `extern uint8 ride_adjacent_station_max_distance;` in header and `uint8 ride_adjacent_station_max_distance = 5;` in C code, but for changes to this variable to work, you'd need to make sure it gets synced properly in network code.

Reply:
It should not be modified, no. Probably a define would be more appropriate. I am just not used to that as a convention. Should the define be all caps?

Reply:
`DEFINES_ARE_WRITTEN_LIKE_THIS`

Reply:
Yes, defines should pretty much always be all caps.

Reply:
`const` in C++ means immutable / read-only, not constant. Use `const_expr` in C++ or `#define` or `enum` in C.